StaryNight/src/test/java/com.starynight/StaryNightApplicationTests.java

```diff
@@ -12,6 +13,7 @@
 class StaryNightApplicationTests {
 
 	//Variables
+	@Autowired
```

Comment:
Autowired the saveService object in the application tests (set up service for autowiring in previous commit)